### PR TITLE
Adds week of month

### PR DESCRIPTION
**Pull request by isonr**

### CODE REVIEW

File: lib/getters/weeks-of/month.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const getMonth = require('../month')
+const getYear = require('../year')
+const getDayOfMonth = require('../day-of/month')
+
+module.exports = function getWeekOfMonth (date) {
+  assert(date)
+
+  const fullYear = getYear(date)
+  const month = getMonth(date)
+  const firstDay = new Date(fullYear, month, 1).getDay()
```
Comment (karimsa) [inline lib/getters/weeks-of/month.js]: There's a util for this; `getStartOf(date, 'month')`.

Comment (isonr) [inline lib/getters/weeks-of/month.js]: Could probably create a util to get first day of month

File: lib/getters/weeks-of/month.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  return Math.ceil((getDayOfMonth(date) + firstDay) / 7)
```
Comment (karimsa) [inline lib/getters/weeks-of/month.js]: First day will always be 1 - is this just adding 1 to the day of month?

File: lib/index.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ exports.getStartOf = require('./getters/start-of')
 
 exports.getDayOfWeek = require('./getters/day-of/week')
 exports.getDayOfMonth = require('./getters/day-of/month')
+exports.getWeekOfMonth = require('./getters/weeks-of/month')
```
Comment (karimsa) [inline lib/index.js]: Rename the directory from `weeks-of` to `week-of`

File: lib/getters/weeks-of/month.js (hunk 1 above)
Comment (isonr) [reply lib/getters/weeks-of/month.js]: 👍

File: test/getters/weeksof/month.test.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const test = require('ava')
+const $ = require('../../utils/constants')
+const Tick = require('../../../')
+
+test('should get weekOfMonth from date', t => {
+  t.is(Tick.getWeekOfMonth($.TODAY), 2, 'today is the second week of the month')
+  t.is(Tick.getWeekOfMonth($.END_OF_JAN_2017), 5, 'end of jan is the fifth week of the month')
+  t.is(Tick.getWeekOfMonth($.FEB_1_1980), 1, 'feb 1 is the first week of the month')
+})
```
Comment (karimsa) [inline test/getters/weeksof/month.test.js]: Tests look good

File: lib/getters/week-of/month.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/**
+ * @file lib/weeks-of/month.js
+ * @license MIT
+ * @copyright 2018-present Foko Inc. All rights reserved.
+ */
+const assert = require('../../assert')
```
Comment (karimsa) [inline lib/getters/week-of/month.js]: Newline after doc comment

**Pull request merged**